src/wp-admin/edit-form-blocks.php

```diff
@@ -72,6 +72,9 @@ static function ( $classes ) {
 	sprintf( '%s/autosaves?context=edit', $rest_path ),
 	'/wp/v2/settings',
 	array( '/wp/v2/settings', 'OPTIONS' ),
+	'/wp/v2/global-styles/themes/' . get_stylesheet(),
+	'/wp/v2/themes?context=edit&status=active',
+	'/wp/v2/global-styles/' . WP_Theme_JSON_Resolver::get_user_global_styles_post_id() . '?context=edit',
```

Comment:
Looks like this can be `null`?

Reply:
I guess the worst thing that can happen is that the path is invalid, it wouldn't cause a warning.

Reply:
It might be a couple of days due to other priorities but I could tighten this up in a follow-up if needed.

Reply:
I am still seeing a pretty significant flash right now, even with the code that was back-ported to RC3.